uptane-standard.md

```diff
@@ -460,45 +470,45 @@ A repository's Root metadata distributes the public keys of the top-level Root,
 
 Targets metadata can also contain metadata about delegations, allowing one Targets role to delegate its authority to another. This means that an individual Targets metadata file might contain only metadata about delegations, only metadata about images, or some combination of the two. The details of how ECUs traverse the delegation tree to find valid metadata about images is specified in {{resolve_delegations}}.
 
-#### Metadata about Images {#targets_images_meta}
+#### Metadata about images {#targets_images_meta}
 
-To be available to install on clients, all images on the repository MUST have their metadata listed in a targets role.  Each targets role MAY provide a list of some images on the repository.  This list MUST provide, at a minimum, the following information about each image:
+To be available to install on clients, all images on the repository MUST have their metadata listed in a Targets role.  Each Targets role MAY provide a list of some images on the repository.  This list MUST provide, at a minimum, the following information about each image:
 
 * The image filename
 * The size of the image in bytes
 * One or more hashes of the image file, along with the hashing function used
 
 ##### Custom metadata about images
 
-In addition to the required metadata, Targets metadata files MAY contain extra metadata for images on the repository. This metadata can be customized for a particular use case. Examples of use cases for different types of custom metadata can be found in the Deployment Considerations document. However, there are a few important pieces of custom metadata that SHOULD be present in most implementations. In addition, there is one element in the custom metadata that MUST be present in the targets metadata from the director.
+In addition to the required metadata, Targets metadata files MAY contain extra metadata for images on the repository. This metadata can be customized for a particular use case. Examples of use cases for different types of custom metadata can be found in the Deployment Considerations document. However, there are a few important pieces of custom metadata that SHOULD be present in most implementations. In addition, there is one element in the custom metadata that MUST be present in the Targets metadata from the Director.
 
-Custom metadata also MAY contain a field or section that is demarcated as custom metadata that MUST match whenever two pieces of metadata are checked against each other--most commonly, when targets metadata from the Director and Image repositories are checked against each other.
+Custom metadata also MAY contain a field or section that is demarcated as custom metadata, and this field MUST match whenever two pieces of metadata are checked against each other. Most commonly, this occurs when targets metadata from the Director and Image repositories are checked against each other.
```

Comment:
Not all custom metadata must match. The must match field/section would be a subsection of the custom metadata (although I agree this was a bit confusing as written before).

Reply:
I went back and forth as to how to fix this. I avoided edits for the most part in this pass except where the syntax was confusing. This sentence WAS confusing, because it is saying that within custom metadata there is a field called custom metadata. I thought my rewrite, which noted that only "this field" --meaning the added field or section, was what had to match, made the distinction you mentioned. If you think you can rephrase the idea in a clearer manner, please do so.

Reply:
Yeah, neither of these changes are ok.

* The first sentence needs to be reverted; you've turned a dependent clause ([...] custom metadata **_that_** MUST match [...]) into an independent clause ([...] custom metadata, **_and_** this field MUST match), and that changes the meaning of the sentence in a way that is somewhere between unclear and incorrect.
* The referent of "this occurs" in the second sentence can't be gleaned easily from context--is "this" the existence of the demarcated field, or the checking of metadata from two sources against each other? Normally, without further clarification (or grammatical disambiguation), we expect a referent to refer to the subject of the previous sentence, which would lead to the wrong conclusion in this case. If you think about it thoroughly, you can figure out what the referent is supposed to be, but you can only get there if you already understand what's going on.

Reply:
Ok. Initially I had two problems with these two sentences. The first is the use of the phrase "custom metadata" twice in the sentence, seemingly referring to a large section and a subsection  simultaneously. Secondly, the addition separating the two phrases with a dash struck me here as sloppy and less than clear. But, I understand your point about accidentally changing the meaning by changing the nature of the clause. So, let me try this another way. What if I take out the repeated reference to "custom metadata"?

Custom metadata MAY also contain a demarcated field or section that MUST match whenever two pieces of metadata are checked against each other, such as when Targets metadata from the Director repository is checked against metadata from the Image repository.

Reply:
Fixed. Let me know if this addresses your concerns.